webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+	entry: './formatter/index.js',
+	output: {
+		path: path.resolve(__dirname, './backend/static/js/formatter/'),
```

Comment:
Why use an absolute path via `path.resolve` for one but not the other?

Reply:
Fair point,
the `path.resolve` was part of the default config.
I'll use it for both.